README.md

```diff
@@ -184,39 +173,43 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-If you like this project, please give it a ⭐️.
+> Write a message to encourage readers to support your project
+
```

Comment:
**[Optional]**

- [ ] It would be nice if you remove this section description to make your readme look clean and professional 😊 These section descriptions are just instructions for you when you will make your readme file and after making your readme file you are supposed to remove all of them because these are unnecessary for the viewers of your repository 🤔